TransbankSDK.sln

```diff
@@ -14,6 +14,8 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
 		README.md = README.md
 	EndProjectSection
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "ConsoleIntegrationTest", "ConsoleIntegrationTest\ConsoleIntegrationTest.csproj", "{0073916B-62AF-4CFF-9F32-0FC6610E6F89}"

```

Comment:
If you add a project that you won't commit, then you shouldn't commit this file with the reference to that project.